lib/src/upgrader.dart

```diff
@@ -497,7 +542,10 @@ class Upgrader {
           ));
     }
     return AlertDialog(
-      title: Text(title),
+      title: Text(
+        title,
+        textAlign: TextAlign.center,
```

Comment:
Can you show me in the Material Design spec where this applies?
https://material.io/components/dialogs#anatomy

Reply:
This for my personal project and should not have added it to the PR

Reply:
I will change it back to what you originally did

Reply:
I just made a new commit to reset it to the default